### PR TITLE
Add Bridgetownrb to topics

### PR DESCRIPTION
**Pull request by cjilbert504**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> A new choice for a static site generator powered by Ruby and webpack aware. Easily add and create plugins to expand site functionality.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of your proposed changes.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/bridgetownrb/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+aliases: bridgetownrb
+created_by: Jared White
+display_name: Bridgetown
+github_url: https://github.com/bridgetown
+logo: bridgetown.png
```
Comment (andrewmcodes) [inline topics/bridgetownrb/index.md:6]: According to the test failure, this needs to be the same name as the topic.

**Review by andrewmcodes — CHANGES_REQUESTED** [review]

File: topics/bridgetownrb/index.md (hunk 1 above)
Comment (cjilbert504) [reply topics/bridgetownrb/index.md]: Got it sorted out now.

**Review by mateo1920 — APPROVED** [review]

**Review by kenyonj — CHANGES_REQUESTED** [review]
Thoughts on using the github/bridgetown avatar for the image instead? This image looks odd with this aspect ratio

https://avatars1.githubusercontent.com/u/63275815?s=200&v=4

**Comment by cjilbert504** [comment]
didn't mean to click that merge button

**Review by may365-media — APPROVED** [review]

**Review by kenyonj — APPROVED** [review]
Looks great. Thanks for your contribution!

**Pull request merged**